### PR TITLE
gitlab-improvement

### PR DESCRIPTION
**Pull request by emamihe**
## Pull request description 

gitlab ci manifest improvement

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: docs/docs/articles/gitlab.md [hunk 1]
```diff
@@ -83,25 +87,39 @@ This workflow establishes a connection to EKS cluster and creates and runs a tes
...
+setup-aws:
   stage: setup
   image: 
     name: amazon/aws-cli
     entrypoint: ["/bin/sh", "-c"]
   script:
-    - yum install -y jq tar gzip 
-    - curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
-    - install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+    - mkdir -p $CI_PROJECT_DIR/tmp/kubeconfig
```
Comment (rangoo94) [inline docs/docs/articles/gitlab.md:101]: Should `kubeconfig` be a directory?

File: docs/docs/articles/gitlab.md [hunk 2]
```diff
@@ -83,25 +87,39 @@ This workflow establishes a connection to EKS cluster and creates and runs a tes
...
+      - $CI_PROJECT_DIR/tmp/kubeconfig
+    expire_in: 1 hour
+
+run-testkube:
+  stage: test
+  image: 
+    name: kubeshop/testkube-cli
+    entrypoint: ["/bin/sh", "-c"]
+  script:
+    - export KUBECONFIG=$CI_PROJECT_DIR/tmp/kubeconfig/config
```
[the remark this answers is not in the document]
Comment (rangoo94) [reply docs/docs/articles/gitlab.md:117]: Thanks, what do you think the about `KUBECONFIG_FILE` variable for simplicity?

[the remark this answers is not in the document]
Comment (rangoo94) [reply docs/docs/articles/gitlab.md:117]: None, if it won't exist - it won't setup `kubeconfig` at all

[the remark this answers is not in the document]
Comment (rangoo94) [reply docs/docs/articles/gitlab.md:117]: Why would we need it then?

[the remark this answers is not in the document]
Comment (rangoo94) [reply docs/docs/articles/gitlab.md:117]: But if you will pass the contents as environment variable to the image, you won't need to set `KUBECONFIG` at all - the image would use `KUBECONFIGFILE` contents.

**Comment by rangoo94** [comment]
> @rangoo94 you agree now to merge it?

In my opinion, we can easily do better (i.e. with `KUBECONFIGFILE` thing, which most likely could be implemented in less than 1 hour, definitely few hours at most).

On the other side, the PR is still better than the current version, as it works, so I'm fine with merging it.

**Pull request merged**